src/frontend/src/pages/FinancePage/ReimbursementRequestPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -0,0 +1,156 @@
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            container
+            mt={2}
+            ml={2}
+            sx={{ backgroundColor: totalCostBackgroundColor, borderRadius: '10px' }}
+          >
+            <Grid item xs={6} textAlign={'center'}>
+              <Typography fontSize={50}>Total Cost</Typography>
+            </Grid>
+            <Grid xs={6} sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+              <Typography fontSize={50}>{`$${reimbursementRequest.totalCost}`}</Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+      </>
+    );
+  };
+
+  const ReimbursementProductsView = () => {
+    const uniqueWbsElementsWithProducts = new Map<String, ReimbursementProduct[]>();
+    reimbursementRequest.reimbursementProducts.forEach((product) => {
+      const wbs = wbsPipe(product.wbsNum);
+      if (uniqueWbsElementsWithProducts.has(wbs)) {
+        const products = uniqueWbsElementsWithProducts.get(wbs);
+        products?.push(product);
+      } else {
+        uniqueWbsElementsWithProducts.set(wbs, [product]);
+      }
+    });
+
+    const keys = [];
+    for (const key of uniqueWbsElementsWithProducts.keys()) {
+      keys.push(key);
+    }
+
+    return (
+      <>
+        <Typography variant="h5">Products</Typography>
+        <Table component={Paper}>
+          <TableHead>
+            <TableRow>
+              <TableCell>WBS Element</TableCell>
+              <TableCell>Products</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {keys.map((key) => {
+              return (
+                <TableRow>
+                  <TableCell>{key}</TableCell>
+                  <TableCell>
+                    {uniqueWbsElementsWithProducts.get(key)?.map((product) => {
+                      return <Chip label={`${product.name} $${product.cost}`} />;
+                    })}
+                  </TableCell>
+                </TableRow>
+              );
+            })}
+          </TableBody>
+        </Table>
+      </>
+    );
+  };
+
+  const ReceiptsView = () => {
+    return (
+      <Box sx={{ maxHeight: `250px`, overflow: 'auto' }}>
+        <Typography variant="h5">Receipts</Typography>
+        {reimbursementRequest.receiptPictures.map((receipt) => {
+          return (
+            <iframe
+              style={{ height: `200px`, width: '50%' }}
+              src={`https://drive.google.com/file/d/${receipt.googleFileId}/preview`}
+              title="ollie"
+            />
+          );
+        })}
+      </Box>
+    );
+  };
+
+  return (
+    <PageLayout
+      title={`${fullNamePipe(reimbursementRequest.recipient)} - ${datePipe(new Date(reimbursementRequest.dateOfExpense))}`}
+      previousPages={[]}
+    >
+      <Grid container spacing={2} mt={2}>
+        <Grid item lg={6} xs={12}>
+          <BasicInformationView />
+        </Grid>
+        {/*Divider*/}
```

Comment:
?

Reply:
Ucant put comments in react so i made a comment here

Reply:
I think if you're gonna do that, it makes more sense to go ahead and put this box in a helpercomponent above and call it as a FinanceDivider or smthn